src/store/authentication/user-reducer.ts

```diff
@@ -0,0 +1,71 @@
+import { createSlice, PayloadAction } from "@reduxjs/toolkit";
+import { deleteCookie, setCookie } from "cookies-next";
+
+import { api } from "../../services/api";
+import { userSignIn, userSignUp } from "./user-action";
+
+const initialState = {
+  loading: false,
+  userInfo: {
+    user: null,
+    isLoggedIn: null,
```

Comment:
Let's remove that as we don't want to keep 2 places in sync. The `useUserInfo` should be the source of truth